x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.tsx

```diff
@@ -243,7 +244,7 @@ const FlyoutRiskSummaryComponent = <T extends RiskScoreEntity>({
                   css={css`
                     position: absolute;
                     right: 0;
-                    top: -${euiThemeVars.euiSize};
+                    top: -${euiTheme.size};
```

Comment:
Blocking: This token does not exist. `euiSize` maps to `euiTheme.size.base`

📖 [Size docs](https://eui.elastic.co/#/theming/sizing/values#scale), [JSON token migration table](https://github.com/elastic/kibana/issues/199715#issuecomment-2546323208)